shardingsphere-sql-parser/shardingsphere-sql-parser-binder/src/main/java/org/apache/shardingsphere/sql/parser/binder/statement/dml/UpdateStatementContext.java

```diff
@@ -42,16 +42,16 @@
     
     public UpdateStatementContext(final UpdateStatement sqlStatement) {
         super(sqlStatement);
-        TableExtractUtils utils = new TableExtractUtils();
+        TableExtractor utils = new TableExtractor();
```

Comment:
Please rename variable name `utils` because of there is no any relationship with variable name and class type.

Reply:
Ok